packages/alert/Alert.tsx

```diff
@@ -46,13 +58,17 @@ export class Alert extends Component<AlertProps> {
 
     return [
       <style>{styles}</style>,
-      isOpen && <div className={className}>
-        <Button close onClick={this.close}>×</Button>
-        <slot />
-      </div>
+      isOpen && (
```

Comment:
Right, this pattern will be enforced by TS lint.
Its just how JS works, if I would omit the brace, JS compiler will on the line `isOpen &&` ( in this case it would trigger an error ) end. by using brace we are telling it that this is not the end of execution, and thanks to that it allow us to format JSX on  separate line which makes the code more readable )